api/types/src/convert.rs

```diff
@@ -47,9 +48,13 @@ use std::{
     convert::{TryFrom, TryInto},
     iter::IntoIterator,
     rc::Rc,
+    str::FromStr,
     sync::Arc,
 };
 
+static OBJECT_MODULE: Lazy<Identifier> = Lazy::new(|| Identifier::from_str("object").unwrap());
```

Comment:
Can't you remove the lazy and make it IdentStr

Reply:
isn't the normal approach to add this type of things to the types crate?
where every one of these is done this way :P

Reply:
Looks like someone made `ident_str!` allowing them as compile time constants